classes/User.php

```diff
@@ -88,6 +90,9 @@ public function getUpdateDetails($this_page, $user) {
 
             $details["url"] = trim(get_http_var("url"));
 
+            $details["can_annotate"] = get_http_var("can_annotate") == "true" ? true : false;
```

Comment:
I think this needs to be further down the file behind the `($this_page == "otheruseredit")` check.